common-java-libs/org.wso2.apk.apimgt.rest.api.devportal.v1.common/build.gradle

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+plugins {
+    alias libs.plugins.openapi.generator.plugin
+}
+
+description = 'WSO2 APK Devportal API Common Component'
+
+sourceSets {
+    main {
+        java {
+            srcDirs("src/gen/java")
+        }
+    }
+}
+
+dependencies {
+    implementation project(':org.wso2.apk.apimgt.api')
+    implementation project(':org.wso2.apk.apimgt.impl')
+    implementation project(':org.wso2.apk.apimgt.rest.api.util')
+    implementation project(':org.wso2.apk.apimgt.user')
+    implementation libs.jackson.databind
+    implementation libs.commons.io
+    implementation libs.commons.lang
+    implementation libs.commons.lang3
+    implementation 'org.apache.commons:commons-collections4:4.4'
```

Comment:
Let's use the entry from the versions toml.

Reply:
resolved